src/syscall/js/js.go

```diff
@@ -292,6 +293,7 @@ func (v Value) Get(p string) Value {
 	return r
 }
 
+//go:noescape
```

Comment:
Justification: this will affect whether strings with dynamic contents but statically known length to not escape anymore.